pkg/translator/prometheusremotewrite/metrics_to_prw_v2_test.go

```diff
@@ -27,7 +27,7 @@ func TestFromMetricsV2(t *testing.T) {
 	want := func() map[string]*writev2.TimeSeries {
 		return map[string]*writev2.TimeSeries{
 			"0": {
-				LabelsRefs: []uint32{1, 2},
+				LabelsRefs: []uint32{1, 2, 3, 4, 5, 6, 7, 8},
```

Comment:
I can't comment on a line that wasn't changed, but should we change line 42 to check the content of `symbolsTable` instead of only checking if it's not nil?

Reply:
Not sure how to test it though, would be nice if we could expose `desymbolizeLabels`  from https://github.com/prometheus/prometheus/blob/main/prompb/io/prometheus/write/v2/symbols.go#L76, so we could compare we get the same labels back. It covered by tests in prometheus too. https://github.com/jmichalek132/prometheus/blob/main/prompb/io/prometheus/write/v2/symbols_test.go#L24

Reply:
Even if we do that, it will take a while until the next Prometheus release. Is there anything we could do now?

Reply:
Ah figured we can do this for now. Is this good enough for now? https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/35751/commits/e758fbbe13457424877017687e59004e418aed21